pages/common/ocamlopt.md

```diff
@@ -0,0 +1,8 @@
+# ocamlopt
+
+> The OCaml native-code compiler.
+> Produces native executables out of OCaml files.
+
+- Create named executable from source file:
+
+`ocamlopt -o {{output_name}} {{source_file.ml}}`
```

Comment:
Could we tweak the values in the token syntax please? Just to make it more consistent with other pages.
```suggestion
`ocamlopt -o {{path/tooutput_file}} {{path/to/source_file.ml}}`
```

Also, are there any other examples we could include in this page? Seems a bit short with just the one.

Reply:
Just updated your comment as it had a missing slash, hope you don't mind. 🙂

Reply:
Thanks, @pxgamer :heart:

Reply:
Let's name it to path/to/binary to be clear.

Reply:
Updated again